hops/featurestore_impl/featureframes/FeatureFrame.py

```diff
@@ -260,11 +261,15 @@ def write_featureframe(self):
               :ValueError: if the user supplied a write mode that is not supported
         """
         print("write feature frame, write_mode: {}".format(self.write_mode))
-        if (self.write_mode == constants.SPARK_CONFIG.SPARK_APPEND_MODE):
+        # TODO (davit): tfrecord format supports append mode
+        if (self._spark_tfrecord_format == constants.FEATURE_STORE.TRAINING_DATASET_TFRECORDS_FORMAT
```

Comment:
Don't you have this check already in line 902 of core.py?

Reply:
You are right, will correct

Reply:
So in core it checks when dataframe is created. This in addition checks when `insert_into_training_dataset` is used. However, as I wrote in other PRs not sure if its it really necessary. What do you think?